bin/strap.sh

```diff
@@ -323,6 +330,13 @@ if git ls-remote "$DOTFILES_URL" &>/dev/null; then
   logk
 fi
 
+# Uninstall non-Brew hostess
+if [ "$GOPATH/go/bin/hostess" -ef /usr/local/bin/hostess ]; then
```

Comment:
is this a common problem?

Reply:
@jtrinklein See note 3.

Reply:
ah, yeah homebrew wasn't (and still isn't) listed as an option for install of hostess, thats why it was installed with go